RecoMuon/TrackerSeedGenerator/interface/SeedMvaEstimator.h

```diff
@@ -0,0 +1,46 @@
+#ifndef RecoMuon_TrackerSeedGenerator_SeedMvaEstimator_h
+#define RecoMuon_TrackerSeedGenerator_SeedMvaEstimator_h
+
+#include "DataFormats/TrajectorySeed/interface/TrajectorySeed.h"
+#include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
+#include "DataFormats/TrajectorySeed/interface/PropagationDirection.h"
+#include "DataFormats/TrajectoryState/interface/PTrajectoryStateOnDet.h"
+#include "DataFormats/TrajectoryState/interface/LocalTrajectoryParameters.h"
+
+#include "DataFormats/L1Trigger/interface/Muon.h"
+
+#include "DataFormats/RecoCandidate/interface/RecoChargedCandidate.h"
+#include "DataFormats/RecoCandidate/interface/RecoChargedCandidateFwd.h"
+
+#include <memory>
+
+class GBRForest;
+
+namespace edm {
+  class FileInPath;
+}
+
+class SeedMvaEstimator {
+public:
+  SeedMvaEstimator(const edm::FileInPath& weightsfile,
+                   const std::vector<double>& scale_mean,
+                   const std::vector<double>& scale_std,
+                   const bool isFromL1);
+  ~SeedMvaEstimator();
+
+  double computeMva(const TrajectorySeed&,
+                    const GlobalVector&,
+                    const l1t::MuonBxCollection&,
+                    int minL1Qual,
```

Comment:
In analogy to what you did for `isFromL1,` then also `minL1Qual` can be defined once for all in the constructor

Reply:
Thanks for the quick review!
I added a new commit now.